tox.ini

```diff
@@ -29,10 +29,7 @@ src = src
 commands_pre = solc-select use 0.8.24 --always-install
 
 commands =
-    fname8 {[testenv:framework]src}
-    isort {[testenv:framework]src} --check --diff
-    black {[testenv:framework]src} --check --diff
-    flake8 {[testenv:framework]src}
+    ruff check {[testenv:framework]src} --fix --show-fixes
```

Comment:
I think this should be changed to `--no-fix`, because if the fix is applied during tox verifications, it will not propagate back to the code base.